Apigee/SmartDocs/Revision.php

```diff
@@ -301,12 +306,15 @@ public function toArray($verbose = TRUE)
      *
      * @param OrgConfig $config
      * @param string $modelId
+     * @param string $modelName
      */
-    public function __construct(OrgConfig $config, $modelId)
+    public function __construct(OrgConfig $config, $modelId, $modelName = "")
```

Comment:
Great!  I haven't seen this done before, can we talk about what you have in mind?  I'd be happy to implement it.